rust/pkg/cardano_serialization_lib.js.flow

```diff
@@ -1506,6 +1526,30 @@ declare export class DNSRecordSRV {
    */
   record(): string;
 }
+/**
+ */
+declare export class DataCost {
+  free(): void;
+
+  /**
+   * !!! DEPRECATED !!!
+   * Since babbage era we should use coins per byte. Use `.new_coins_per_byte` instead.
+   * @param {BigNum} coins_per_word
+   * @returns {DataCost}
+   */
+  static new_coins_per_word(coins_per_word: BigNum): DataCost;
+
+  /**
+   * @param {BigNum} coins_per_byte
+   * @returns {DataCost}
+   */
+  static new_coins_per_byte(coins_per_byte: BigNum): DataCost;
+
+  /**
+   * @returns {BigNum}
+   */
+  get_cost_per_byte(): BigNum;
```

Comment:
Please rename `coins_per_byte()`

1. No `get` for consistency with other getters in all the structures
2. Use `coins` for consistency with constructors